app/lib/screens/app_onboarding/preview.dart

```diff
@@ -66,15 +66,17 @@ class Preview {
     // delete study subscription and progress
     if (containsQueryPair('cmd', 'reset')) {
       if (selectedStudyObjectId != null) {
-        final StudySubject subject = await SupabaseQuery.getById<StudySubject>(
-          selectedStudyObjectId!,
-          selectedColumns: [
-            '*',
-            'study!study_subject_studyId_fkey(*)',
-            'subject_progress(*)',
-          ],
```

Comment:
We should still delete the related subject_progress data, when deleting a study_subjects